ports/onnxruntime/portfile.cmake

```
@@ -0,0 +1,149 @@
+        # training  onnxruntime_ENABLE_TRAINING_OPS
+        cuda      onnxruntime_USE_CUDA
+        cuda      onnxruntime_USE_CUDA_NHWC_OPS
+        openvino  onnxruntime_USE_OPENVINO
+        tensorrt  onnxruntime_USE_TENSORRT
+        tensorrt  onnxruntime_USE_TENSORRT_BUILTIN_PARSER
+        directml  onnxruntime_USE_DML
+        directml  onnxruntime_USE_CUSTOM_DIRECTML
+        winml     onnxruntime_USE_WINML
+        coreml    onnxruntime_USE_COREML
+        mimalloc  onnxruntime_USE_MIMALLOC
+        valgrind  onnxruntime_USE_VALGRIND
+        xnnpack   onnxruntime_USE_XNNPACK
+        nnapi     onnxruntime_USE_NNAPI_BUILTIN
+        azure     onnxruntime_USE_AZURE
+        llvm      onnxruntime_USE_LLVM
+        test      onnxruntime_BUILD_UNIT_TESTS
+        test      onnxruntime_BUILD_BENCHMARKS
+        test      onnxruntime_RUN_ONNX_TESTS
+        framework onnxruntime_BUILD_APPLE_FRAMEWORK
+        framework onnxruntime_BUILD_OBJC
+        nccl      onnxruntime_USE_NCCL
+        mpi       onnxruntime_USE_MPI
+        minimal   onnxruntime_ORT_MINIMAL_BUILD
+    INVERTED_FEATURES
+        abseil    onnxruntime_DISABLE_ABSEIL
+        cuda      onnxruntime_USE_MEMORY_EFFICIENT_ATTENTION
+)
+
+if("python" IN_LIST FEATURES)
+    x_vcpkg_get_python_packages(
+        PYTHON_VERSION 3
+        PACKAGES numpy sympy
+        OUT_PYTHON_VAR PYTHON3
+    )
+    get_filename_component(PYTHON_PATH "${PYTHON3}" PATH)
+else()
+    vcpkg_find_acquire_program(PYTHON3)
+    get_filename_component(PYTHON_PATH "${PYTHON3}" PATH)
+endif()
+message(STATUS "Using python3: ${PYTHON3}")
+vcpkg_add_to_path(PREPEND "${PYTHON_PATH}")
+
+# see tools/ci_build/build.py
+vcpkg_cmake_configure(
+    SOURCE_PATH "${SOURCE_PATH}/cmake"
+    OPTIONS
+        ${FEATURE_OPTIONS}
+        -DPython_EXECUTABLE:FILEPATH=${PYTHON3}
+        -DProtobuf_PROTOC_EXECUTABLE:FILEPATH=${PROTOC}
+        -DCMAKE_DISABLE_FIND_PACKAGE_Git=ON
+        -DBUILD_PKGCONFIG_FILES=${BUILD_SHARED}
+        -Donnxruntime_BUILD_SHARED_LIB=${BUILD_SHARED}
+        -Donnxruntime_BUILD_WEBASSEMBLY=OFF
+        -Donnxruntime_CROSS_COMPILING=${VCPKG_CROSSCOMPILING}
+        -Donnxruntime_USE_FULL_PROTOBUF=OFF # minimalize protoc execution
+        -Donnxruntime_USE_PREINSTALLED_EIGEN=ON
+        -Donnxruntime_USE_EXTENSIONS=OFF
+        -Donnxruntime_USE_NNAPI_BUILTIN=${VCPKG_TARGET_IS_ANDROID}
+        -Donnxruntime_ENABLE_CPUINFO=ON
+        -Donnxruntime_ENABLE_MICROSOFT_INTERNAL=OFF
+        -Donnxruntime_ENABLE_BITCODE=${VCPKG_TARGET_IS_IOS}
+        -Donnxruntime_ENABLE_PYTHON=OFF
+        -Donnxruntime_ENABLE_EXTERNAL_CUSTOM_OP_SCHEMAS=OFF
+        -Donnxruntime_ENABLE_LAZY_TENSOR=OFF
+        -Donnxruntime_NVCC_THREADS=1 # parallel compilation
+        -Donnxruntime_DISABLE_RTTI=OFF
+        -Donnxruntime_USE_NEURAL_SPEED=OFF
+        -DUSE_NEURAL_SPEED=OFF
+        # for ORT_BUILD_INFO
+        -DORT_GIT_COMMIT:STRING="v${VERSION}"
+        -DORT_GIT_BRANCH:STRING="v${VERSION}"
```

Unless the quotes are to become part of the value:
```suggestion
        "-DORT_GIT_COMMIT:STRING=v${VERSION}"
        "-DORT_GIT_BRANCH:STRING=v${VERSION}"
```